packages/mobile/src/web3/saga.ts

```diff
@@ -173,6 +194,43 @@ function* assignDataKeyFromPrivateKey(key: string) {
   yield put(setPrivateCommentKey(privateCEK))
 }
 
+function getPrivateKeyFilePath(account: string): string {
```

Comment:
Is this the long term strategy for pk management or just a stopgap?

Reply:
The long-term is one step beyond this. Right now, I am writing the plain-text key to the private storage, I would like to encrypt it as well with the `pincode` which we are storing in the key store.